docs/openapi/howto/archboard_feedback.md

```diff
@@ -154,4 +155,28 @@ At a glance:
     "x-ms-pageable": {
        "nextLinkName": null
     }
- ```
+```
+
+## Operation should be LRO
+
+You will get this feedback if your operation is a Long Running Operation (LRO), and follow the LRO protocol (usually using Location, Operation-Location, etc.). Those operations need to be tagged as LRO in the Swagger, as this makes SDK generates a specific return type as a poller.
+
+```json
+"/myservice/subscriptions" :
+  "put": {
+    "operationId": "CreateVirtualMachine",
+    "x-ms-long-running-operation": true
```

Comment:
Link to `x-ms-long-running-operation` docs?

Reply:
Added link to autorest doc